docs/docs/api/rest/README.md

```diff
@@ -1222,6 +1272,331 @@ curl "https://api.pipedream.com/v1/webhooks/wh_abc123" \
 
 Pipedream exposes a limited API for retrieving workflow events, and plans to expose APIs for creating, updating, and deleting workflows programmatically in the future. If you'd like to see this, please upvote and add detail on your use case on [this GitHub issue](https://github.com/PipedreamHQ/pipedream/issues/8735).
 
+### Create a workflow
+
+---
+
+Creates a new workflow within an organization's project. This endpoint allows defining workflow steps, triggers, and settings, based on a supplied template.
+
+#### Endpoint
+
+```
+POST /workflows
+```
+
+#### Request Body
+
+`org_id` **string**
+
+[Switch to your workspace's context](/workspaces/#switching-between-workspaces) and [find your org's ID](/workspaces/#finding-your-workspace-s-id).
+
+`project_id` **string**
+
+The ID of the project where the new workflow will be created. To find your project ID, switch to your desired worksapce, and click on Projects in the top left of the Pipedream dashboard.
+
+Click on the project where you'd like to create the new workflow, and the project ID can be found in the URL, starting with `proj_`.
+
+If the URL is [https://pipedream.com/@pd-testing/projects/proj_GzsRY5N/tree](https://pipedream.com/@pd-testing/projects/proj_GzsRY5N/tree), your `project_id` is `proj_GzsRY5N`.
+
+`template_id` **string** 
+
+The ID of the workflow template to base the workflow on. To find a workflow's `template_id`, navigate to your workflow that you'd like to create a template for, and click "Create share link". If the URL created is [https://pipedream.com/new?h=tch_Vdfl0l](https://pipedream.com/new?h=tch_Vdfl0l), your `template_id` is `tch_Vdfl01`.
+
+`steps` **object**: Definitions of the steps to include in the workflow. Each key within the steps object represents a step type, and the associated `props` define the step's configuration.
+
+`triggers` **array**: Definitions of the triggers that will start the workflow. Each item in the array represents a trigger, with its type and `props`.
+
+`settings` **object**: Additional settings for the workflow, such as `name` and `auto_deploy`.
+
+#### Example Request
+
+```json
+{
+  "project_id": "proj_wx9sgy",
+  "org_id": "o_BYDI5y",
+  "template_id": "tch_3BXfWO",
+  "steps": {
+    "code": {
+      "props": {
+        "stringProp": "asdf"
+      }
+    },
+    "keyauth_hello_world": {
+      "props": {
+        "keyauth": {
+          "authProvisionId": "apn_Nb6h9v"
+        }
+      }
+    }
+  },
+  "triggers": [
+    {},
```

Comment:
```suggestion
    {},
```
@malexanderlim should we remove these empty trigger objects? Or was that intentional?

Reply:
Sorry, meant to remove those.